docs/_includes/common/filter-menu.html

```diff
@@ -0,0 +1,541 @@
+    <form name="filter-menu-form-1">
+        <div class="filter-menu-form__items">
+            <div class="filter-menu-form__item">
+                <span class="checkbox">
+                    <input aria-label="filter menu checkbox example" class="checkbox__control" type="checkbox" name="filter-menu-form-1-checkbox-item-1" />
+                    <span class="checkbox__icon" hidden>
+                        <svg class="checkbox__unchecked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-unchecked"></use>
+                        </svg>
+                        <svg class="checkbox__checked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-checked"></use>
+                        </svg>
+                    </span>
+                </span>
+                <span>Item 1 with a long string that will wrap to another line</span>
+            </div>
+            <div class="filter-menu-form__item">
+                <span class="checkbox">
+                    <input aria-label="filter menu checkbox example" class="checkbox__control" type="checkbox" name="filter-menu-form-1-checkbox-item-2" />
+                    <span class="checkbox__icon" hidden>
+                        <svg class="checkbox__unchecked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-unchecked"></use>
+                        </svg>
+                        <svg class="checkbox__checked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-checked"></use>
+                        </svg>
+                    </span>
+                </span>
+                <span>Item 2</span>
+            </div>
+            <div class="filter-menu-form__item">
+                <span class="checkbox">
+                    <input aria-label="filter menu checkbox example" class="checkbox__control" type="checkbox" name="filter-menu-form-1-checkbox-item-3" />
+                    <span class="checkbox__icon" hidden>
+                        <svg class="checkbox__unchecked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-unchecked"></use>
+                        </svg>
+                        <svg class="checkbox__checked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-checked"></use>
+                        </svg>
+                    </span>
+                </span>
+                <span>Item 3</span>
+            </div>
+        </div>
+        <button type="submit" class="filter-menu-form__footer">Apply</button>
+    </form>
+</span>
+    {% endhighlight %}
+    </div>
+
+    <h3 id="filter-menu-form-footer">Filter Menu With Scroll</h3>
+    <p>When the filter menu reaches more than 50% of the height of the page, the items will begin to scroll.</p>
```

Comment:
Do we need to do this 50% height restriction for the non-button version of the menu? Is that in the spec?

Reply:
Well, the spec does say that there is a "max height" of 50% of the page. Therefore, I'm assuming yes. Let me confirm.

Reply:
It only makes sense for the dropdown version in my opinion.

Reply:
Added comments to move to `filter-menu-button` and commented the example to be used there as well.